hrm-domain/hrm-core/permissions/initializeCanForRules.ts

```diff
@@ -149,6 +149,8 @@ const setupAllow = <T extends TargetKind>(
   );
 
   return (performer: TwilioUser, target: any) => {
+    // Let system users do anything for now, might need to tighten this up in future
+    if (performer.isSystemUser) return true;
```

Comment:
We are currently limiting this by the endpoints it can access. If we are going to limit access to internal endpoints once it's called from a lambda then that will lock this access down still further. Whilst it's good to triple check, I don't want system permissions to become a source of otherwise avoidable bugs